examples/traffic-light.manifold

```diff
@@ -0,0 +1,41 @@
+* = import Manifold.core
+* = import Manifold.digital
+
+Type TrafficLight = (
+  green:Boolean  = false,
+  yellow:Boolean = false,
+  red:Boolean    = false
+)
+
+GREEN_SECONDS  = 10
+YELLOW_SECONDS = 3
+TOTAL_SECONDS  = 2 * (GREEN_SECONDS + YELLOW_SECONDS)
+
+seconds = Manifold.Digital.count(hz: 1, mod: TOTAL_SECONDS)
+
+Manifold.Digital.if(
+  (seconds < GREEN_SECONDS, -> {
+    north_light = south_light = (green: true)
```

Comment:
You're right. The grammar is ambiguous and I don't have a good answer for how to fix it. Lets just keep that on the DL at symposium ```\me pretends this is a private repo```